tests/unit/specs/SpectrumAnalyser.spec.js

```diff
@@ -1,28 +1,30 @@
 import { mount } from '@vue/test-utils';
-import Component from '@/components/SpectrumAnalyser';
+import SpectrumAnalyser from '@/components/SpectrumAnalyser';
 
 describe('SpectrumAnalyser', () => {
   test('is a Vue instance', () => {
-    const wrapper = mount(Component);
+    const wrapper = mount(SpectrumAnalyser);
     expect(wrapper.isVueInstance()).toBeTruthy();
   });
 
   it('emits an error and doesn’t render when initialization fails', () => {
-    const wrapper = mount(Component);
+    const wrapper = mount(SpectrumAnalyser);
     expect(wrapper.emitted().error).toBeTruthy();
     expect(wrapper.html()).toBeFalsy();
   });
 
   it('renders when there’s a valid spectrum', () => {
-    const wrapper = mount(Component);
+    const wrapper = mount(SpectrumAnalyser);
     wrapper.setData({ spectrum: new Uint8Array(null) });
-    expect(wrapper.html()).toMatch(/canvas/);
+    expect(wrapper.emitted().error).toBeFalsy();
```

Comment:
This assertions always fails, because `navigator.mediaDevices.getUserMedia()` does not exist in jsdom. I think line 19 and line 27 have to be reverted.

Reply:
:heavy_check_mark: